inert/dynamic-inert-on-focused-element.tentative.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>Dynamic inertness on focused element</title>
+<link rel="author" title="Oriol Brufau" href="mailto:obrufau@igalia.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/interaction.html#focus-fixup-rule">
+<meta name="assert" content="If a focused element becomes expressly inert, it stops being focused.">
+<div id="log"></div>
+
+<input class="has-focus becomes-inert">
```

Comment:
has-focus is slightly confusing, when I read the markup, it makes me think the elements themselves are focused.

How about `focusable` or `initial-focus`?

Reply:
Used `check-focus`, to make it clear that this is the thing that defines the tests.